packages/gatsby-plugin-manifest/src/gatsby-ssr.js

```diff
@@ -31,13 +31,25 @@ exports.onRenderBody = ({ setHeadComponents }, pluginOptions) => {
   }
 
   // Add manifest link tag.
-  headComponents.push(
-    <link
-      key={`gatsby-plugin-manifest-link`}
-      rel="manifest"
-      href={withPrefix(`/manifest.webmanifest`)}
-    />
-  )
+  if (pluginOptions.crossOrigin) {
```

Comment:
you should be able to do this without the if else

```
headComponents.push(
  <link
    key={`gatsby-plugin-manifest-link`}
    rel="manifest"
    href={withPrefix(`/manifest.webmanifest`)}
    crossOrigin={pluginOptions.crossOrigin}
  />
)
```

React will omit any falsy variables so link will discard crossOrigin when it's undefined|false|null|0